services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -31,20 +41,71 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
           _ shouldBe VersioningException("Could not create a repository for deleted project.")
         }
       }
+
       "return new active Project with 201 response" taggedAs Service in {
         val project = projectWithRepo
         when(request(activeProject)).thenReturn(Future.successful(Response(StatusCodes.Created.intValue, "", Map())))
         gitLabProjectVersioning.createRepository(activeProject).map {
           _ shouldBe Right(project)
         }
       }
+
       "throw new VersioningException with not 400 response" taggedAs Service in {
         when(request(activeProject)).thenReturn(Future.successful(Response(StatusCodes.BadRequest.intValue, "", Map())))
         gitLabProjectVersioning.createRepository(activeProject).map {
           _ shouldBe Left(VersioningException("The repository was not created. Response status: 400"))
         }
       }
     }
+
+    "updateFile" should {
+      "return create file response when file is new" taggedAs Service in {
+        val path = URLEncoder.encode(newFile.path.toString, "UTF-8")
+
+        when(
```

Comment:
You can use whenReady and there will be no Future in shouldBe

Reply:
Yes, but what the point of doing this?

Reply:
Because we use AsyncWordSpec, in your case we can use just WordSpec.

Reply:
Ok, will change but it's arguable

Reply:
And I don't have any Futures in shouldBe. More than that, if you will look at your tests, you have the same methods for resolving this.

Reply:
@AndreiMaksimenko , @GrigoriiBerezin  `whenReady` looks cleaner, let's use it.

Reply:
At least, it's not cleaner for me: whenReady isn't mock for me, it's just fact, that Future returns some result, like Andrei did at his createRepository tests.